crates/sui-bridge/src/metrics.rs

```diff
@@ -344,14 +344,14 @@ impl BridgeMetrics {
             .unwrap(),
             auth_agg_ok_responses: register_int_counter_vec_with_registry!(
                 "bridge_auth_agg_ok_responses",
-                "Total number of ok respones from auth agg",
+                "Total number of ok response from auth agg",
```

Comment:
This and the next should be plural as "total number" suggests a sum typically greater than one.